src/average.js

```diff
@@ -12,6 +12,19 @@
     - average([1, '2']) // Retorno: undefined;
 */
 
-const average = () => {};
+const average = (arrayNumeros) => {
+  let totalNumeros = arrayNumeros.length;
+  let sum = 0;
+  if (arrayNumeros.length < 1) {
+    return undefined;
+  }
+  for (let number of arrayNumeros) {
+    if (typeof (number) === 'string') {
+      return undefined;
+    }
+    sum += number;
+  }
+  return Math.round(sum / totalNumeros);
+};
```

Comment:
Jeff, você concorda que o código fica mais legível quando separamos algumas seções pulando uma linha, ex:

```suggestion
const average = (arrayNumeros) => {
  const totalNumeros = arrayNumeros.length;
  let sum = 0;
  
  if (arrayNumeros.length < 1) {
    return undefined;
  }
  
  for (let number of arrayNumeros) {
    if (typeof (number) === 'string') {
      return undefined;
    }
    
    sum += number;
  }
  
  return Math.round(sum / totalNumeros);
};
```